src/app/games/kamisado/tests/kamisado.component.spec.ts

```diff
@@ -48,6 +47,7 @@ describe('KamisadoComponent', () => {
         expect(componentTestUtils.getComponent().chosen.isAbsent()).toBeTrue();
     }));
     it('should allow to pass if stuck position', fakeAsync(async() => {
+        // Given a board with a stuck piece being the one that has to play
```

Comment:
that has to move or thas has to be played

Reply:
gedone